deploy/charts/emqx-operator/templates/webhook-validating-configuration.yaml

```diff
@@ -2,11 +2,11 @@
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
-  {{- if (index .Values "cert-manager" "enable") }}
+   {{- if (index .Values "cert-manager" "enable") }}
   annotations:
     cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ include "emqx-operator.fullname" . }}-serving-cert
-  {{- end }}
-  name: {{ include "emqx-operator.fullname" . }}-validating-webhook-configuration
+  {{- end}}
+  name: {{ include "emqx-operator.fullname" . }}-mutating-webhook-configuration
```

Comment:
This is `webhook-validating-configuration.yaml` file, but you add `mutating-webhook-configuration `